### PR TITLE
Bootstrap test suite

### PR DESCRIPTION
**Pull request by gianarb**
Hi! This module has not suite test, why?
I don't know the development's story but in my opinion beyond a maintainable problem this stuff is a visibility problem.
I cannot cover all module not but maybe together we can surmount this lack.

Thanks to a good test coverage we can works at new features without worry to break all..

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -47,5 +51,10 @@
         "classmap": [
             "./Module.php"
         ]
+    },
+    "autoload-dev": {
```
Comment (Ocramius) [inline composer.json]: Alignment to be fixed

File: tests/OptionsTest.php [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+namespace ZendDeveloperToolsTest;
+
+use PHPUnit_Framework_TestCase;
+use ZendDeveloperTools\Options;
+
+class OptionsTest extends PHPUnit_Framework_TestCase
+{
+    public function testStatusOfDefaultConfiguration()
+    {
+        $dist = require __DIR__."/../config/zenddevelopertools.local.php.dist";
```
Comment (Ocramius) [inline tests/OptionsTest.php]: Why do you need real options? Put the array directly here

**Comment by Ocramius** [comment]
Please remove the `composer.lock`

File: tests/OptionsTest.php (hunk 2 above)
Comment (gianarb) [inline tests/OptionsTest.php]: This is only an idea but if a future change broken the default conf? We can
use a data provider to tests other stuff in other tests..

2015-03-27 11:35 GMT+01:00 Marco Pivetta <notifications@github.com>:

> In tests/OptionsTest.php
> <https://github.com/zendframework/ZendDeveloperTools/pull/179#discussion_r27286412>
> :
>
> > @@ -0,0 +1,17 @@
> > +<?php
> > +namespace ZendDeveloperToolsTest;
> > +
> > +use PHPUnit_Framework_TestCase;
> > +use ZendDeveloperTools\Options;
> > +
> > +class OptionsTest extends PHPUnit_Framework_TestCase
> > +{
> > +    public function testStatusOfDefaultConfiguration()
> > +    {
> > +        $dist = require __DIR__."/../config/zenddevelopertools.local.php.dist";
>
> Why do you need real options? Put the array directly here
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zendframework/ZendDeveloperTools/pull/179/files#r27286412>
> .
>



-- 
Gianluca Arbezzano
www.gianarb.it

**Comment by gianarb** [comment]
@Ocramius can I add travis.yml in this PR? Are you interested to enable it for this project?

File: tests/ModuleTest.php [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+use PHPUnit_Framework_TestCase;
+use PHPUnit_Framework_Assert;
+use PHPUnit_Framework_MockObject_MockObject;
+use ZendDeveloperTools\Module;
+use DoctrineModuleTest\ServiceManagerTestCase;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Input\InputInterface;
+/**
+ * @author Martin Keckeis <martin.keckeis1@gmail.com>
+ * @covers \DoctrineModule\Module
```
Comment (samsonasik) [inline tests/ModuleTest.php]: DoctrineModule?

Comment (gianarb) [inline tests/ModuleTest.php]: Fixed

**Comment by gianarb** [comment]
In my opinion the test suite is implemented..
Now we can require testing for new feature and we can collaborate for a good coverage..

**Comment by Ocramius** [comment]
Merging as-is: please consider sending a second PR with the suggested changes

**Pull request merged**